packages/ng/src/app/select/select.directive.ts

```diff
@@ -0,0 +1,257 @@
+import {
+	OnChanges,
+	Directive,
+	OnInit,
+	Input,
+	Output,
+	EventEmitter,
+	forwardRef,
+	OnDestroy,
+	Renderer2,
+	ElementRef,
+	SimpleChanges,
+	ViewContainerRef,
+	HostListener,
+	HostBinding
+} from '@angular/core';
+import {
+	Overlay,
+	OverlayConfig,
+} from '@angular/cdk/overlay';
+import {
+	NgModel,
+	ControlValueAccessor,
+	NG_VALUE_ACCESSOR,
+	NG_VALIDATORS,
+	Validator,
+	ValidatorFn,
+	ValidationErrors,
+	AbstractControl
+} from '@angular/forms';
+import { Subscription } from 'rxjs/Subscription';
+
+import { LuPopoverTrigger } from '../popover';
+import { LuSelectPopover } from './select.popover.component';
+import { LuSelectOption } from './select.option.component';
+
+/** KeyCode for End Key */
+const END = 'End';
+/** KeyCode for Home Key */
+const HOME = 'Home';
+/** KeyCode for Escape Key */
+const ESCAPE = 'Escape';
+/** KeyCode for Backspace Key */
+const BACKSPACE = 'Backspace';
+/** KeyCode for Delete Key */
+const DELETE = 'Delete';
+/** KeyCode for Enter Key */
+const ENTER_KEY = 'Enter';
+/** KeyCode for ArrowUp Key */
+const UP_KEY = 'ArrowUp';
+/** KeyCode for ArrowDown Key */
+const DOWN_KEY = 'ArrowDown';
+
+/**
+ * Directive to put on a div to allow it to react with a popover to emulate a select component
+ */
+@Directive({
+	selector: 'div[luSelect]',
+	providers: [
+		{ provide: NG_VALUE_ACCESSOR, useExisting: forwardRef(() => LuSelectDirective), multi: true },
+		{ provide: NG_VALIDATORS, useExisting: forwardRef(() => LuSelectDirective), multi: true },
+	],
+})
+export class LuSelectDirective<T>
+extends LuPopoverTrigger
+implements ControlValueAccessor, OnDestroy, OnChanges, OnInit,  Validator {
```

Comment:
There is indeed lots of common with luApiPicker because i use this code to start the component.

I have in mind to refactor in a near future the luApiPicker to mutualise more code. As i said in my previous comment, luApiPicker = autocomple and luSelect = select, so I have to create a luAutocomplete that will share code and logic between the two and use this new class as parent class for luApiPicker.

So for the moment, is "normal" if there some duplication of code. Consider this as a Work In Progress. and i have in mind you comment ;)